source/_components/image_processing.facebox.markdown

```diff
@@ -58,3 +65,29 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+## {% linkable_title Automations %}
+
+Use the `image_processing.detect_face` events to trigger automations, and breakout the `trigger.event.data` using a [data_template](https://www.home-assistant.io/docs/automation/templating/). The following example automation sends a notification when Ringo Star is recognised:
+
+{% raw %}
+```yaml
+- id: '1120092824666'
+  alias: Ringo Starr recognised
+  trigger:
+  - event_data:
+      name: 'Ringo_Starr'
+    event_type: image_processing.detect_face
+    platform: event
+  condition: []
+  action:
+  - data_template:
+      message: Ringo_Starr recognised with probability {{ trigger.event.data.confidence }}
+      title: Door-cam notification
+    service: notify.platform
+```
+{% endraw %}
+
+## {% linkable_title Optimising resources %}
+
+[Image-classifier components](https://www.home-assistant.io/components/image_processing/) process the image from a camera at a fixed period given by the `scan_interval`. This leads to excessive processing if the image on the camera hasn't changed, as the default `scan_interval` is 10 seconds. You can override this by adding to your config `scan_interval: 10000` (setting the interval to 10,000 seconds), and then call the `image_processing.scan` service when you actually want to perform processing. 
```

Comment:
`Image-classifier components` -> `Image processing components`

Reply:
On second thoughts, perhaps this should be placed in https://www.home-assistant.io/components/image_processing/ as it applies to all `image_processing` components..?

Reply:
ok